source/Calamari.Common/Features/Scripting/Bash/Bootstrap.sh

```diff
@@ -307,6 +307,70 @@ function decrypt_and_parse_variables {
     exec 3<&-
 }
 
+# -----------------------------------------------------------------------------
+# Functions to report Kubernetes Manifests via service message
+#	Accepts 2 arguments:
+#	  string: the namespace
+#	  string: the manifest
+# -----------------------------------------------------------------------------
+function report_kubernetes_manifest 
+{
+  MANIFEST="$1"
+  NAMESPACE="$2"  
+  
+  LINES=()
+
+  # Read lines from file into the array
+  while IFS= read -r LINE; do
+    LINES+=("$LINE")
+  done <<< "$MANIFEST"
+
+  MANIFESTS=()
+  
+  CURRENT=""
+  for LINE in "${LINES[@]}"; do
+    if [ "$(echo "$LINE" | xargs)" = "---" ]; then
+      MANIFESTS+=("$CURRENT")
+      CURRENT=""
+    else
+      CURRENT="$CURRENT$LINE\n"	
+    fi
+  done
+    
+  MANIFESTS+=("$CURRENT")
+  
+  for i in "${MANIFESTS[@]}"; do
```

Comment:
naming nit since it's not an index

```suggestion
  for manifest in "${MANIFESTS[@]}"; do
```

Reply:
Fixed in [c97b400](https://github.com/OctopusDeploy/Calamari/pull/1474/commits/c97b40030024beab06c3eef03b273687266986da)